omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/UserResource1_8.java

```diff
@@ -205,7 +205,8 @@ public UserAndPassword1_8 save(UserAndPassword1_8 user) {
 	 */
 	@Override
 	public UserAndPassword1_8 getByUniqueId(String uuid) {
-		return new UserAndPassword1_8(Context.getUserService().getUserByUuid(uuid));
+		User user = Context.getUserService().getUserByUuid(uuid);
```

Comment:
Is there a reason we need to rerun the query here as opposed to: `return user != null ? new UserNameAndPassword1_8(user) : null`?